src/foundry/client-esm/nue/tour.d.mts

```diff
@@ -1,9 +1,276 @@
+import type { ValueOf } from "fvtt-types/utils";
+
 /**
  * A Tour that shows a series of guided steps.
- * @remarks TODO: Stub
  */
-declare class Tour {}
+declare class Tour {
+  /**
+   * Construct a Tour by providing a configuration.
+   * @param config  - The configuration of the Tour
+   * @param options - Additional options for configuring the tour
+   *                  (default: `{}`)
+   */
+  constructor(config: Tour.Config, options?: Tour.ConstructorOptions);
+
+  static STATUS: Readonly<{
+    UNSTARTED: "unstarted";
+    IN_PROGRESS: "in-progress";
+    COMPLETED: "completed";
+  }>;
```

Comment:
Nit: I'd move this into the `Tour` namespace. I have a policy that any object gets to be an `interface`.
1. It helps reusability.
2. It allows people to extend this with their own extra stuff. It can be really unclear where someone _might_ extend something so it can be useful.